info.go

```diff
@@ -182,3 +182,18 @@ func scanFdInfoReader(r io.Reader, fields map[string]interface{}) error {
 
 	return nil
 }
+
+// EnableStats starts the measuring of the runtime (in ns)
```

Comment:
This reads a bit vaguely to me. I'm not familiar with this particular feature, so I immediately had the following questions:
- Which programs get their runtime stats collected? Surely not all BPF programs on the machine, that's why this syscall was introduced.
- How can the stats be queried?
- What am I supposed to pass to `which`?

Reply:
> Which programs get their runtime stats collected? Surely not all BPF programs on the machine, that's why this syscall was introduced.

As far as I can tell, at the moment there is only the option to enable the colleciton of statistics. If enabled statistics for all executing eBPF programs are collected.
I can imagine, this statistic collection will be extendend and maybe more specific at some point in the future. 

> What am I supposed to pass to which?
At the moment, there is only a single valid value for `which` - `BPF_STATS_RUN_TIME`. `which` was introduced because of discussions in https://github.com/cilium/ebpf/pull/213#discussion_r574313592. 
As `BPF_STATS_RUN_TIME` is provided by golang.org/x/sys/unix I did not add an additional wrapper.